src/manual-gimbap/index.js

```diff
@@ -0,0 +1,46 @@
+// Webfonts
+import { getIframeId, getWebfonts } from '../_shared/js/messages.js';
+
+getIframeId().then(() => {
+    getWebfonts();
+});
+
+// Adds quotes to title if set in DFP.
```

Comment:
All this can be wrapped in a call to `write` (see in `_shared/js/dom.js`) to perform DOM mutations inside the animation frame callback.